client/src/Components/JournalCard/JournalCardStyle.js

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components/macro';
+
+export const JournalCard = styled.div`
+  width: 16rem;
+  height: auto;
+  margin: auto 40px auto 0;
+  box-shadow: 0 0 5px lightgray;
+  position: relative;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
```

Comment:
I thought we were using 'Roboto'. We need consistency or create a theme and use variables